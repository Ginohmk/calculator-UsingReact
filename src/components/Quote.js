/* eslint-disable react/jsx-one-expression-per-line */
import React from 'react';

export default function Quote() {
  return (
    <div className="quote">
      <p>
        Mathematics is not about numbers,equations,computation or algorithm: it
        is about understanding. -<span>William Paul Thurston</span>
      </p>
    </div>
  );
}
