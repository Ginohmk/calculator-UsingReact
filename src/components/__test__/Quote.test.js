import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Test for Home component', () => {
  test('Home snapShot test', () => {
    const tree = renderer.create(<Quote />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
