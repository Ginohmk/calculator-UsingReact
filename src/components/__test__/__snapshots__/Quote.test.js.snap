// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Home component Quote snapShot test 1`] = `
<div
  className="quote"
  data-testid="quote-data"
>
  <p>
    Mathematics is not about numbers,equations,computation or algorithm: it is about understanding. -
    <span>
      William Paul Thurston
    </span>
  </p>
</div>
`;
